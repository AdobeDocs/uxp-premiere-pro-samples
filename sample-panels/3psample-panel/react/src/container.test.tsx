/*************************************************************************
 * ADOBE CONFIDENTIAL
 * ___________________
 *
 *  Copyright 2023 Adobe
 *  All Rights Reserved.
 *
 * NOTICE:  All information contained herein is, and remains
 * the property of Adobe and its suppliers, if any. The intellectual
 * and technical concepts contained herein are proprietary to Adobe
 * and its suppliers and are protected by all applicable intellectual
 * property laws, including trade secret and copyright laws.
 * Dissemination of this information or reproduction of this material
 * is strictly forbidden unless prior written permission is obtained
 * from Adobe.
 **************************************************************************/

import renderer from "react-test-renderer";
import Content from "./content";
import Header from "./header";

describe("Content", () => {
  it("should render correctly with valid Content and theme state", () => {
    let component = renderer.create(
      <div>
        <Header />
        <Content message={["Test"]} />
      </div>
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
