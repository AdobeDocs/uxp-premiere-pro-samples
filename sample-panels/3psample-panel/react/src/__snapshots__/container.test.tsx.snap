// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content should render correctly with valid Content and theme state 1`] = `
<div>
  <div
    className="plugin-header"
  >
    Scripting Plugin
  </div>
  <style>
    
    .plugin-header {
      color:#ccc;
      background-color: #333;
      width: 100%;
      padding: 5px;
      font-size: 1.6em;
      padding-bottom: 8px;
    }
  
  </style>
  <div
    className="plugin-content"
  >
    <div>
      Test
    </div>
  </div>
  <style>
    
    .plugin-content {
      color:#fff;
      background-color: #444;
      width: 100%;
      padding: 8px;
      font-size: 1.2em;
      min-height: 15em;
      border-radius: 5px;
    }
  
  </style>
</div>
`;
